packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -357,37 +357,46 @@ function WorklistDataGrid(props) {
     }
   };
 
-  const selectMultipleRows = useCallback((newEndRowIndex, rangeInfo) => {
-    if (!onRowSelect || !rangeInfo) {
-      return;
+  const getRowsInRangeToUnselect = useCallback((startOfRange, prevEndOfRange, newEndOfRange) => {
+    let rowIdsToUnselect = [];
+    if (!prevEndOfRange) {
+      // short-circuit since there was no previous range, there is nothing to unselect.
+      return rowIdsToUnselect;
+    }
+
+    if (startOfRange > prevEndOfRange) {
```

Comment:
Aren't these row indexes?  The variable names could probably be a little more clear.

Reply:
Since we are no longer passing these as parameter, multiSelectRange ref is directly used.